frontend/packages/console-app/src/components/guided-tours/TaskHeader.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from 'react';
+import cx from 'classnames';
+import { Title } from '@patternfly/react-core';
+import { TaskState } from './utils/guided-tour-status';
+import './QuickStartSection.scss';
+
+type TaskHeaderProps = {
+  title: string;
+  taskIndex: number;
+  subtitle?: string;
+  taskState?: TaskState;
+  size?: 'md' | 'lg' | 'xl' | '2xl' | '3xl' | '4xl';
+};
+
+const TaskHeader: React.FC<TaskHeaderProps> = ({ title, taskIndex, subtitle, taskState, size }) => {
+  const TaskIcon: React.FC = () => {
+    switch (taskState) {
+      case TaskState.SUCCESS:
+        return (
+          <span className="co-icon-and-text__icon">
+            <i className="fas fa-check-circle" />
+          </span>
+        );
+      case TaskState.FAILED:
+        return (
+          <span className="co-icon-and-text__icon">
+            <i className="fas fa-times-circle" />
```

Comment:
Use patternfly icons.

Reply:
picked this from patternfly v4 icons https://www.patternfly.org/v4/design-guidelines/styles/icons#updated-icon-recommendations
couldn't find a pf-icon class for this